src/kasse.c

```diff
@@ -112,177 +112,75 @@ static void print_screen(void) {
 static void print_log(char *name, int item_price, int einheiten, char *nickname,
                       char *rest) {
   char *time = get_time();
-  char price[10];
-  /* Format:
-     Transaction-ID (Anzahl verkaufter Einträge, inklusive des zu druckenden!)
-     -- 6-stellig
-     Uhrzeit -- 8-stellig
-     Eintragname (= Getränk) -- 9-stellig
-     Preis (in Cents) -- 9-stellig
-     Anzahl -- 2-stellig
-     Nickname (falls es vom Guthaben abgezogen wird) -- 10-stellig
-     restguthaben (9-stellig)
-
-     + 7 leerzeichen
-     --> 48 zeichen
-     */
-  if (format_euro(price, 10, item_price) == NULL) {
+  /* TODO program a custom character for the eurosign.
+   * Until then, we just overwrite it with "EUR" */
+  char price[EUR_FORMAT_MINLEN + 2];
+  if (format_euro(price, sizeof(price), item_price) == NULL) {
     cprintf("Preis %d konnte nicht umgerechnet werden\r\n", item_price);
     exit(1);
   }
+  price[EUR_FORMAT_MINLEN - 1] = 'E';
+  price[EUR_FORMAT_MINLEN + 0] = 'U';
+  price[EUR_FORMAT_MINLEN + 1] = 'R';
 
-  sprintf(print_buffer, "%c[%3u] %s - %-" xstr(
-                            MAX_ITEM_NAME_LENGTH) "s - %s - %s - %d - an %s\r",
+  /* TODO hack */
+  rest[EUR_FORMAT_MINLEN - 1] = 'E';
+
+  /* clang-format off */
+  sprintf(print_buffer,
+          /*  Transaction-ID (Anzahl verkaufter Einträge, inklusive des zu druckenden!)
+              -- 6-stellig (Lüge!) */
```

Comment:
Why is that comment a lie? Can we rectify the comment instead of labeling it as wrong?